sensors/lego_sensor_class.c

```diff
@@ -122,6 +123,11 @@
  *   error. The values are fixed point numbers, so check `decimals` to see if
  *   you need to divide to get the actual value.
  * .
+ * `text_value` (read-only)
+ * : Returns a string representing sensor-specific text value. The string may
+ *   contain embedded line feed characters, is limited to PAGE_SIZE bytes
```

Comment:
All of our other attributes return space-separated lists of values. I don't thing adding newlines other than the usual one at the end is a good idea.